store/sqlstore/group_store.go

```diff
@@ -1035,3 +1045,81 @@ func (s *SqlGroupStore) GetGroups(page, perPage int, opts model.GroupSearchOpts)
 
 	return groups, nil
 }
+
+func (s *SqlGroupStore) ifGroupsThenTeamUsersRemovedQuery(teamID string, groupIDs []string, isCount bool) squirrel.SelectBuilder {
+	var selectStr string
+
+	if isCount {
+		selectStr = "count(DISTINCT Users.Id)"
+	} else {
+		tmpl := "Users.*, TeamMembers.SchemeGuest, TeamMembers.SchemeAdmin, TeamMembers.SchemeUser, %s AS GroupIDs"
+		if s.DriverName() == model.DATABASE_DRIVER_MYSQL {
+			selectStr = fmt.Sprintf(tmpl, "group_concat(UserGroups.Id)")
+		} else {
+			selectStr = fmt.Sprintf(tmpl, "string_agg(UserGroups.Id, ',')")
+		}
+	}
+
+	groupIDsSQL := strings.Join(groupIDs, "', '")
+
+	query := s.getQueryBuilder().Select(selectStr).
+		From(`TeamMembers
+			JOIN Teams ON Teams.Id = TeamMembers.TeamId
+			JOIN Users ON Users.Id = TeamMembers.UserId
+			LEFT JOIN Bots ON Bots.UserId = TeamMembers.UserId
+			JOIN GroupMembers ON GroupMembers.UserId = Users.Id
+			JOIN UserGroups ON UserGroups.Id = GroupMembers.GroupId`).
+		Where(fmt.Sprintf(`TeamMembers.DeleteAt = 0
```

Comment:
This is a weird mix where you mix squirrel with Sprintf, why not keep using squirrel here?